src/commands/server/object.ts

```diff
@@ -83,7 +89,48 @@ const NewCommand: ExtendedGluegunCommand = {
       return undefined;
     }
 
-    const { props, refsSet, schemaSet } = await server.addProperties({ objectsToAdd, referencesToAdd });
+    // Parse property arguments from CLI or prompt interactively
```

Comment:
The following code, which generates the props, is redundant. Please move it to a method and use the method here instead.